bazel/grpc_python.bzl

```diff
@@ -0,0 +1,72 @@
+"""Custom rules for gRPC Python"""
+
+
+# Adapted with modifications from tensorflow/tensorflow/core/platform/default/build_config.bzl
+# Native Bazel rules don't exist yet to compile Python, but rules have been written at
```

Comment:
"Native Bazel rules don't yet exist to compile Python (`link to issue in Bazel issue tracker`)" - so that anyone reading this comment can click the link and see whether or not they have been added.